framework/db/ActiveRecord.php

```diff
@@ -207,7 +207,12 @@ protected static function findByCondition($condition)
     protected static function filterCondition(array $condition)
     {
         $result = [];
+        // valid column names are table column names or column names prefixed with table name
```

Comment:
both column names prefixed with table name and not prefixed with table name are valid